### PR TITLE
Add DANIMESQ space

### PR DESCRIPTION
**Pull request by DaniellMesquita**
DANIMESQ is my personal token I'll use for offchain governance.

Signed-off-by: Daniell Mesquita <daniellmesquita@protonmail.com>

### CODE REVIEW

**Comment by DaniellMesquita** [comment]
@e-asia

File: spaces/daniell-mesquita/skin.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  --primary-color: #f74174;
+--bg-color: #f74174;
+--text-color: #fff;
+--link-color: #fff;
+--heading-color: #fff;
+--border-color: #04b7bc;
+--header-bg: #04b7bc;
+--block-bg: transparent;
+  }
+}
```
Comment (e-asia) [inline spaces/daniell-mesquita/skin.scss:11]: Scss indentation is not correct, you have to delete the extra quote. "}"
```suggestion
.daniell-mesquita {
  --primary-color: #f74174;
  --bg-color: #f74174;
  --text-color: #fff;
  --link-color: #fff;
  --heading-color: #fff;
  --border-color: #04b7bc;
  --header-bg: #04b7bc;
  --block-bg: transparent;
}
```

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by DaniellMesquita** [comment]
@e-asia ok!

**Comment by e-asia** [comment]
@DaniellMesquita The images file size should not exceed 50KB. Please reduce their weight.

**Comment by DaniellMesquita** [comment]
@e-asia still the same good icon and reduced much bellow 50KB!

**Review by e-asia — APPROVED** [review]

**Pull request merged**